workspaces/linux-host/Makefile

```diff
@@ -78,6 +78,8 @@ veracruz-test: test-dependencies
 $(RUNTIME_ENCLAVE_BINARY_PATH):
 	$(MAKE) -C ../linux-runtime linux
 
+CLIPPY_OPTIONS = --no-deps -A clippy::type_complexity -A clippy::module_inception -D warnings
```

Comment:
Given these are configuration options, can this variable binding be moved to the top of the `Makefile`?  (Same with the other `Makefile`s, too.)